README.md

```diff
@@ -168,7 +168,7 @@ For development, update the previously added line in `DCS World\Scripts\MissionS
 - dofile(lfs.writedir()..[[Scripts\DCS-gRPC\grpc-mission.lua]])
 + GRPC = {
 + 	dllPath = [[C:\Development\DCS-gRPC\rust-server\target\debug\]],
-+ 	luaPath = [[C:\Development\DCS-gRPC\rust-server\lua\]]
++ 	luaPath = [[C:\Development\DCS-gRPC\rust-server\lua\DCS-gRPC]]
```

Comment:
Is this the anticipated suggestion inbound on my PR?

Reply:
Nope, I am going to suggest to:
- remove the settings inside of the `MissionScripting.lua` completely in favor of the settings file we have now
- get rid of the `Data/dcs-grpc.lua` file (also in favor of the settings file we have now)
- integrate the change you've added to the README in your PR into the project (so having a `lua/Hooks/DCS-grpc.lua` which loads settings and then `lua/DCS-gRPC/dcs-hook.lua`; where the latter is then used for actual logic inside of the hooks environment)